pb/transaction.proto

```diff
@@ -65,6 +65,7 @@ message SigChainTxn {
 message RegisterName {
 	bytes  registrant = 1;
 	string name       = 2;
+	uint32 expires_at = 3;
```

Comment:
We should definitely store `expiresAt` in db, but for txn, it's more convenient to use `duration` instead of `expiresAt` so user don't need to know what's the current block height when creating transaction (same as subscribe txn)